tests/test_editable.py

```diff
@@ -0,0 +1,189 @@
+# SPDX-FileCopyrightText: 2021 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import pathlib
+import sys
+
+import pytest
+
+import mesonpy
+
+from mesonpy import _editable
+
+from .test_wheel import EXT_SUFFIX
+
+
+def test_walk():
+    entries = set(_editable.walk(os.fspath(pathlib.Path(__file__).parent.joinpath('packages/complex/complex'))))
```

Comment:
You can use the `package_complex` fixture here. Same for the other tests.

```suggestion
def test_walk(package_complex):
    entries = set(_editable.walk(os.fspath(package_complex.joinpath('complex'))))
```